src/styles/variables.js

```diff
@@ -71,7 +71,7 @@ export const TinyFont = '11px';
 export const MicroFont = '9px';
 
 export const BoldFont = '700';
-export const MediumFontWeight = '600';
+export const MediumFontWeight = '500';
```

Comment:
why was this changed? will this be consistent with our UX designs across all the components and views?

Reply:
Changed back to 600, renamed to `SemiBoldFont`